src/modules/form/components/viewable/DynamicViewGroup.tsx

```diff
@@ -52,6 +57,8 @@ const DynamicViewGroup = (props: ViewGroupItemComponentProps) => {
       );
     case Component.Table:
       return <TableGroup key={props.item.linkId} {...props} viewOnly />;
+    case Component.SignatureGroup:
+      return <SignatureGroupCard key={props.item.linkId} viewOnly {...props} />;
```

Comment:
Updated `DynamicViewGroup` to support read-only display of signature items:

<img width="1024" alt="Screenshot 2024-02-22 at 3 29 27 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/ea3aa0a2-4729-4cc6-9e04-f97747c046b2">